### ISSUE TITLE
Is it possible to move focus to the column options with the keyboard?

### ISSUE THREAD

**Issue by christianp** [gristlabs/grist-core#1146]
The keyboard interaction for tables is really good, but I can't see how to get out of it and use the keyboard to move around the rest of the Grist interface. Is there a way of moving focus from the table to the column options sidebar, with the keyboard?

And once the focus is in the column options, pressing <kbd>Tab</kbd> seems to move focus back to the table and on to the next column. So is the answer "we haven't done that bit yet"?

**Comment by dsagal** [comment]
Yeah, that's the answer. Grist carefully tries to keep focus on the cells as much as possible (so that you can start typing, or navigate, or copy-paste). In principle, you could imagine a shortcut to switch between "Data focus" (on cells) and "UI focus" (on rest of UI), but such UI focus is basically non-existent today -- many elements don't have `tabindex` set, the cell still shows a cursor as if it has focus, etc. So basically, it's quite a lot of work to implement.
